src/markdown.js

```diff
@@ -11,11 +11,34 @@ ${code}
 </stache-code-block>
 `;
 
-  renderer.heading = (text, level) =>
-`<stache-page-anchor level="${level}">
-${text}
+  renderer.heading = (text, level) => {
+    let parsedHeading = '';
+
+    switch (level) {
+      case 1:
+        parsedHeading = `
+<stache-page-title>
```

Comment:
Question. Do we want to allow for this? Since we are handling the page title on the stache tag itself, do we want to potentially have multiple page titles on here? @Blackbaud-SteveBrush @Blackbaud-BrandonJones

Reply:
Hmmmm I suppose that's fair...  Just make it an h1?